packages/components/src/components/combobox/combobox.custom.styles.ts

```diff
@@ -0,0 +1,23 @@
+import { css } from 'lit';
```

Comment:
If we are overriding the combobox.styles, why don´t we add the code here to it?

Reply:
@schilchSICKAG 
Mario mentioned he would find it better, that we vendor the select styles and don`t just copy it initially, because if something is changed in the select styling, we need to keep in mind to change it also in the combobox. 

(we use the exact same styling as for the select)